tests/functional/phpinfo-1.phpt

```diff
@@ -16,7 +16,7 @@ phpinfo();
 mongodb
 
 mongodb support => enabled
-mongodb version => 0.%d.%d
+mongodb version => %d.%d.%d
```

Comment:
I'd just bump it to 1.. We do want to verify the version is atleast realistic..
It'd be embarrassing if the first stable release was 6.6.6